pandas/_libs/src/datetime/np_datetime.c

```diff
@@ -1016,11 +1012,11 @@ int convert_timedelta_to_timedeltastruct(pandas_timedelta_metadata *meta,
 
         // put frac in seconds
         if (td < 0 && td % (1000LL * 1000LL * 1000LL) != 0)
-          frac = td / (1000LL * 1000LL * 1000LL) - 1;
```

Comment:
I *think* we already lint this file (cpplint). is this not picked up? rather have the proper config settings than manually changing things.

Reply:
No argument here, but cpplint is well outside my domain of competence.  I only noticed this indentation when trying to move this func into cython.

Reply:
@WillAyd any experience with cpplint? want to give this a shot, e.g. making sure we are linting the *.c (more/better)?

Reply:
Can't claim experience with cpplint but looking at the bash script that's it appears to already be hitting all of the files in this folder. I ran both versions through cpplint locally and neither version caused any issue with the highest level of verbosity and neither raised an alarm. Surprisingly I couldn't get an error when changing the line to a 1 or 3 space indent either, so I think there might be some bug with cpplint itself.

In looking at Google's style guide they suggest a 2 space indent. To match that and keep consistent with the rest of the file I would actually suggest removing the two spaces added here and remove the two from the indented block after the ``else`` statement instead

https://google.github.io/styleguide/cppguide.html#Conditionals

Reply:
see https://github.com/apache/arrow?files=1

for some hints in how to lint / enforce this as well

Reply:
Are you saying that within arrow cpplint is calling out those whitespace issues? I tried replicating the filters I saw in that project via:

    cpplint --verbose=2 --filter=-whitespace/comments,-readability/todo,-build/header_guard,-build/c++11,-runtime/references,-build/include_order pandas/_libs/src/datetime/np_datetime.c

And then tried an even more verbose version without any filter
    
    cpplint --verbose=1 pandas/_libs/src/datetime/np_datetime.c

And neither made any mention of those lines, whether I set them at 2, 4 or even 3 space indents.

Reply:
no just a guess that they have more options set for linting

Reply:
Looking at the cpplint source, I don't see anything that checks for proper 2-space indentation of conditional statements, nor was there any documentation hinting that it could. The ``whitespace/indentat`` check offered only seems to make sure that you don't start with 1 or 3 spaces, but doesn't do much beyond that. Unfortunately I don't see any programmatic options to continually validate this with that tool as is